packages/core/src/routes/organization/invitations.ts

```diff
@@ -64,6 +68,50 @@ export default function organizationInvitationRoutes<T extends AuthedRouter>(
 
       ctx.body = await organizationInvitations.insert(body, yes(query.skipEmail));
       ctx.status = 201;
+      return next();
+    }
+  );
+
+  router.put(
+    '/:id/status',
+    koaGuard({
+      params: z.object({
+        id: z.string(),
+      }),
+      body: OrganizationInvitations.updateGuard
+        .pick({
+          acceptedUserId: true,
+        })
+        .extend({
+          status: z.enum([
+            OrganizationInvitationStatus.Accepted,
+            OrganizationInvitationStatus.Revoked,
+          ]),
+        }),
+      response: organizationInvitationEntityGuard,
+      status: [200],
```

Comment:
exception status? 422, 404 e.g.

Reply:
this is interesting. 422 can be successfully returned (see tests) while we don't have it here.

Reply:
ah because we don't catch errors in `koaGuard`. i'll update this in a separate pull